docs/examples/pandapower_example.ipynb

```diff
@@ -0,0 +1,634 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PandaPower conversion\n",
+    "\n",
+    "This example illustrates conversion from PandaPower to power-grid-model input data. \n",
+    "We can then calculate power-flow with it or convert to a different formats like PGM JSON."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Load the PandaPower Data\n",
```

Comment:
The notebook is not working. I will create a new branch to propose some fixes.

Reply:
@Laurynas-Jagutis is already taking care of the get function for attributes. But lets merge only the notebook.